scripts/jenkins/Jenkinsfile

```diff
@@ -610,13 +664,13 @@ def checksum(file) {
 
 /* Generate a Stage
```

Comment:
Why is it called `maybe`?

Reply:
I am referring that this should be documented. And "Generate a Stage" is misleading if it maybe generates a stage.

Reply:
it is always generating a stage, the stage might just be skipped